src/main/scala/cromwell/binding/values/WdlFile.scala

```diff
@@ -1,8 +1,14 @@
 package cromwell.binding.values
 
 import java.io.File
+import java.nio.file.Path
 import cromwell.binding.types.WdlFileType
 
+object WdlFile {
+  def apply(value: String): WdlFile = WdlFile(new File(value))
+  def apply(value: Path): WdlFile = WdlFile(value.toFile)
```

Comment:
Sure... if not now then I think it'll be harder later and the consistency will be nice